frontend/.eslintrc.json

```
@@ -22,6 +22,7 @@
     ],
     "rules": {
       "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-      "import/no-unresolved": "off"
+      "import/no-unresolved": "off",
+      "func-names": "off"
     }
```

I have encountered an error please add this line to fix it, same with backend's .eslintrc.json
```suggestion
    },
    "root": true
```

https://stackoverflow.com/questions/40903307/error-cannot-read-config-file-x-package-json/40914623#40914623

---

thank you! just added.